clustered/client/src/main/java/org/ehcache/clustered/client/config/builders/ClusteredStoreConfigurationBuilder.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.clustered.client.config.builders;
+
+import org.ehcache.clustered.client.config.ClusteredStoreConfiguration;
+import org.ehcache.clustered.common.Consistency;
+import org.ehcache.config.Builder;
+
+/**
+ * {@link Builder} for the {@link ClusteredStoreConfiguration}.
+ */
+public class ClusteredStoreConfigurationBuilder implements Builder<ClusteredStoreConfiguration> {
```

Comment:
Configurations for serializers and copiers are exposed directly, no? How is this different from those cache level configurations?